GameData/RealismOverhaul/Filter_Extensions_Configs/non_WIP_RO_FE_config.cfg

```diff
@@ -8,11 +8,32 @@
 {
     @SUBCATEGORIES
     {
+		list = 95,RO-DEPRECATED
         list = 96,WIP-RO
         list = 97,non-RO
     }
 }
 
+//  ==================================================
```

Comment:
Does it make sense to have a category where you can see and use the deprecated parts? The main point of deprecating si to have the parts be removed from circulation over time, so that no crafts are affected when it is finally deleted.

Reply:
I second this, if something is deprecated you probably shouldn't be able to see it at all.

Reply:
The logic is copied from the soft and hard removal of nonRO and nonRP0 parts. If wanted I could strip it down to just always hiding parts with no control but that does make it harder to verify the deprecation is working as intended. I already found and fixed that RO-RD9B was being deprecated when it should not have been.

Because of how FilterExtensions works, its not possible to completely hide unlocked parts. I think moving to another category helps keep them from showing up in the main categories but setting to -1 may be good enough too.

Working on the assumption that :NEEDS[NoRODeprecated] passes because we have added it to the Express Install. When a user upgrades RO/RP-1/etc any unpurchased parts that have been newly deprecated will be moved to ORPHANS and completely hidden. However because KSP saves part purchases into the savegame under the tech tree node they were purchased from, any already purchased parts will remain unlocked. If the user does not have FilterExtensions they will not be able to find the part, but with FilterExtensions it is always possible to find the part if you go digging hard enough through the parts by module or resource type filtering.

Reply:
The use case of finding incorrectly deprecated parts is a good motivation for having something like this. Could it be made more clear that the deprecated category shouldn't be used or something?